src/index.html

```diff
@@ -12,6 +12,88 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+      <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+        <fieldset class="fieldset requireds">
+          <label class="form-field">
+            Email
+            <input type="email" name="email" placeholder="email@example.com" required>
+          </label>
+          <label class="form-field">
+            Password
+            <input 
+            type="password" 
+            name="password" 
+            minlength="5"
+            maxlength="16"
+            required
+            >
```

Comment:
```suggestion
            <input 
              type="password" 
              name="password" 
              minlength="5"
              maxlength="16"
              required
            >
```
indents (everywhere) 😉